Revise README

brotchen21 opened the pull request:
Updated README to remove AI service references and adjust sections for clarity.

brotchen21 committed:
Revise README 

Updated README to remove AI service references and adjust sections for clarity.

brotchen21 committed:
Fix formatting in README.md

Updated the README to improve formatting and clarity.

SENULT reviewed (commented):
Install AI Service Dependencies (Optional)
~~```bash cd ai_service pip install -r requirements.txt cd ..

mi có thể xóa hẳn dòng này trên file readme lun á

brotchen21 committed:
Revise README for clarity and updated instructions

Updated README to reflect changes in features, installation steps, and removed deprecated sections.

brotchen21 commented:
> Install AI Service Dependencies (Optional) ~~```bash cd ai_service pip install -r requirements.txt cd ..
> 
> mi có thể xóa hẳn dòng này trên file readme lun á

t xoá rồi ấy

brotchen21 committed:
Revise README

Updated README to reflect new project name and features.
